src/module/item/container/sheet.ts

```diff
@@ -1,4 +1,16 @@
-import { PhysicalItemSheetPF2e } from "@item/physical/index.ts";
+import { ItemSheetOptions } from "@item/base/sheet/sheet.ts";
+import { PhysicalItemSheetData, PhysicalItemSheetPF2e } from "@item/physical/index.ts";
 import type { ContainerPF2e } from "./document.ts";
 
-export class ContainerSheetPF2e extends PhysicalItemSheetPF2e<ContainerPF2e> {}
+export class ContainerSheetPF2e extends PhysicalItemSheetPF2e<ContainerPF2e> {
+    override async getData(options?: Partial<ItemSheetOptions>): Promise<EquipmentSheetData> {
+        return {
+            ...(await super.getData(options)),
+            isApex: this.item._source.system.traits.value.includes("apex"),
```

Comment:
Can this be done once at the physical sheet level?

Reply:
It could. I asked you about it and you mentioned preferring it per sheet, but I can imagine having a different idea when seeing the actual scope of it.